DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh

```diff
@@ -169,34 +169,42 @@ elif [[ -n "$ALIEN_JDL_USETHROTTLING" ]]; then
   export TIMEFRAME_RATE_LIMIT=1
 fi
 
-echo "[INFO (async_pass.sh)] envvars were set to TFDELAYSECONDS ${TFDELAYSECONDS} TIMEFRAME_RATE_LIMIT ${TIMEFRAME_RATE_LIMIT}"
-
 if [[ ! -z "$ALIEN_JDL_SHMSIZE" ]]; then export SHMSIZE=$ALIEN_JDL_SHMSIZE; elif [[ -z "$SHMSIZE" ]]; then export SHMSIZE=$(( 16 << 30 )); fi
 if [[ ! -z "$ALIEN_JDL_DDSHMSIZE" ]]; then export DDSHMSIZE=$ALIEN_JDL_DDSHMSIZE; elif [[ -z "$DDSHMSIZE" ]]; then export DDSHMSIZE=$(( 32 << 10 )); fi
 
 # root output enabled only for some fraction of the cases
-# keeping AO2D.root QC.root o2calib_tof.root MFTAssessment.root mchtracks.root mchclusters.root
+# keeping AO2D.root QC.root o2calib_tof.root mchtracks.root mchclusters.root
 
-SETTING_ROOT_OUTPUT="ENABLE_ROOT_OUTPUT_o2_mch_reco_workflow= ENABLE_ROOT_OUTPUT_o2_mft_reco_workflow= ENABLE_ROOT_OUTPUT_o2_tof_matcher_workflow= ENABLE_ROOT_OUTPUT_o2_aod_producer_workflow= ENABLE_ROOT_OUTPUT_o2_qc= "
+SETTING_ROOT_OUTPUT="ENABLE_ROOT_OUTPUT_o2_mch_reco_workflow= ENABLE_ROOT_OUTPUT_o2_tof_matcher_workflow= ENABLE_ROOT_OUTPUT_o2_aod_producer_workflow= ENABLE_ROOT_OUTPUT_o2_qc= "
 
 keep=0
 
+if [[ -n $ALIEN_INPUT_TYPE ]] && [[ "$ALIEN_INPUT_TYPE" == "TFs" ]]; then
+  unset WORKFLOW_PARAMETER
+  export WORKFLOW_PARAMETERS=CTF
+  INPUT_TYPE=TF
+  export TPC_CONVERT_LINKZS_TO_RAW=1
```

Comment:
the TPC_CONVERT... depends on the period. We need it for the linkZs data, but no longer since we have the improved linkZs with the fixes for the metadata.

Reply:
What period needed it?

Reply:
out of my mind, it was 10th of August. But you want to know exactly, you have to ask the TPC experts when the firmware with the fix for the metainfo in the ILBZS was deployed.

Reply:
Ok, if it is this update, then I know and I also know the run number. thanks!